.github/workflows/ruff.yml

```diff
@@ -18,4 +18,4 @@ jobs:
     - run: python -Im pip install --user ruff
```

Comment:
would be good to pin this to a matching version to the one in [`.pre-commit-config.yaml`](https://github.com/torchbox/wagtail-footnotes/blob/main/.pre-commit-config.yaml#L21) (with my pref to bump the pre-commit one to 0.5.0